plugin/src/main/java/com/playmonumenta/scriptedquests/quests/components/actions/ActionCommand.java

```diff
@@ -12,6 +12,10 @@
 import com.playmonumenta.scriptedquests.Plugin;
 import com.playmonumenta.scriptedquests.quests.components.QuestPrerequisites;
 
+// For error requesting UUID of null npc
```

Comment:
I suspect the auto input arranger will eventually eat this comment - nothing else documents imports, probably reasonable not to have this